### DISCUSSION TITLE
How to use code-server as a proxy for other machines?

### DISCUSSION THREAD

**Discussion by MeowcaTheoRange** [coder/code-server#6863] [Q&A] [answered]
Hello,
I have a Proxmox setup, with an LXC container running `code-server` on it, in a network of other LXC containers with SSH enabled. All of these containers can communicate and share data together.
What I want to do is use `code-server` as a proxy, connecting to the other containers without having to install `code-server` on each of them manually (as that would require me to manage multiple `code-server` instances).

Kinda like this:

- Log into `code-server` on `code.example.com` 
- Choose from remotes `node1.example.com` or `node2.example.com` 
- `container-code` does SSH into `container-node1`/`container-node2` 
- display to `code-server` web interface.

Thanks!

**Comment by MeowcaTheoRange** [comment]
I have NGINX on each node, so I'm going to just install `code-server` on those nodes and host `code-server` via a path on each node for now, but if anyone can think of anything, I will take it :]

**Reply by code-asher** [reply, marked as answer]
code-server has no support for remote SSH, so installing on each node is the only way for now.

There is an open issue but it is not being worked on: https://github.com/coder/code-server/issues/1315

It is not possible to use the official remote SSH extension either, partially because it is not licensed for use outside of the official VS Code release, and partially because it does not support the web version of VS Code.
